test/factories/users.rb

```diff
@@ -17,6 +17,10 @@
       end
     end
 
+    trait :with_person do
```

Comment:
And should we move this to an association in the main block of the User factory? I think we always want a Person when we create a User, right?

Reply:
There's a test to check the creation of a person for a user that needs a user without a person, if we don't want that test then we can add person to the main block.

Reply:
I think if we are always creating a User with a person then we should update the factories accordingly.